libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -124,11 +125,17 @@ const AP_Param::GroupInfo AP_Vehicle::var_info[] = {
     AP_SUBGROUPINFO(nmea, "NMEA_", 17, AP_Vehicle, AP_NMEA_Output),
 #endif
 
+#if AP_DDS_ENABLED
+    // @Group: XRCE
+    // @Path: ../AP_DDS_Client/AP_DDS_Client.cpp
+    AP_SUBGROUPPTR(xrce_client, "XRCE_", 18, AP_Vehicle, AP_DDS_Client),
```

Comment:
i think this should use a DDS_ parameter prefix

Reply:
Done